configure.ac

```diff
@@ -17,11 +19,8 @@ AC_PROG_SED
 AC_PROG_CPP
 AC_SYS_LARGEFILE
 AM_PROG_CC_C_O
-SQ_PROG_CPP_POSIX_2001
 SQ_PROG_CC_WALL
 
-AC_DEFINE([_POSIX_C_SOURCE], [200112L], [POSIX 2001 compatibility])
-AC_DEFINE([FUSE_USE_VERSION], [26], [Version of FUSE API to use])
```

Comment:
Another discovery issue: Not defining this here will break all non-pkgconfig use of FUSE, since it will never get defined _anywhere_. (That might be OK for FUSE 2, since it defaults to `21` in `fuse.h`, but code using fuse3 is **required** to define a value for `FUSE_USE_VERSION` explicitly before including the header.)

Reply:
Sorry, never mind — I hadn't realized these bits were already merged to `master`.